bot/exts/evergreen/snakes/_snakes_cog.py

```diff
@@ -739,71 +740,62 @@ async def hatch_command(self, ctx: Context) -> None:
     @snakes_group.command(name='movie')
     async def movie_command(self, ctx: Context) -> None:
         """
-        Gets a random snake-related movie from OMDB.
+        Gets a random snake-related movie from TMDB.
 
         Written by Samuel.
         Modified by gdude.
+        Modified by Will Da Silva.
         """
-        url = "http://www.omdbapi.com/"
-        page = random.randint(1, 27)
+        page = random.randint(1, 16)
```

Comment:
What happens if the chosen page is greater than the last page of the search results?

Reply:
Previously the max page was hard-coded, so I figured I'd just go that route for now, and improve it if it's deemed necessary. There are 16 pages in the search results for "snake". In the future, as more movies are added to TMDB, there will eventually be more pages, but that won't break anything here, and I do not expect there will ever be fewer pages than the amount there are now.

This could be fixed by setting the upper range of the random number generation to the `total_pages` field returned by the search, but this would require another request. Alternatively the first request could be limited to 16 pages (or some other low hard-coded number of pages), and then `total_pages` would be used afterwards.

Reply:
Or we could go for some more elaborate pre-fetching/caching.

Reply:
> Alternatively the first request could be limited to 16 pages (or some other low hard-coded number of pages), and then total_pages would be used afterwards.

This sounds good. The initial number should be slightly lower than the threshold to give some leeway for films being removed.